snoop/data/digests.py

```diff
@@ -65,15 +65,40 @@ def filetype(mime_type):
     return None
 
 
+def full_path(file):
+    node = file
+    elements = [file.name]
+    while node.parent:
+        node = node.parent
+        elements.append(node.name)
+    return '/'.join(reversed(elements))
+
+
+def directory_id(directory):
+    return f'_directory:{directory.pk}'
+
+
+def parent_id(file):
+    parent = file.parent
+
+    if isinstance(parent, models.File):
+        return parent.blob.pk
+
+    if isinstance(parent, models.Directory):
+        return directory_id(parent)
+
+    return None
+
+
 def get_document_data(digest):
     with digest.result.open() as f:
         digest_data = json.loads(f.read().decode('utf8'))
 
     first_file = digest.blob.file_set.order_by('pk').first()
-    filename = path = first_file.name
 
     return {
         'id': digest.blob.pk,
+        'parent_id': parent_id(first_file),
```

Comment:
if the "Parent" button only leads you to the first file, then how can you navigate to the other places where a document shows up?

Reply:
You can't yet.